tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -80,7 +80,7 @@ def commandCallback(self, event):
     # Device Properties
     # -----------------
     CspSubarrayNodeFQDN = device_property(
-        dtype='str', default_value="mid-csp/elt/subarray01"
+        dtype='str', default_value="tango://dada:10000/mid-csp/elt/subarray01"
```

Comment:
Remove hard coded tango host name.

Reply:
Yes. It is removed in the latest commits.
Commit ID : [fae6faf5249aeec5d0c60948a4f13a33d70c746d](https://github.com/ska-telescope/tmc-prototype/pull/39/commits/fae6faf5249aeec5d0c60948a4f13a33d70c746d)